src/main/features/ccj/views/their-details.njk

```diff
@@ -17,18 +18,50 @@
         </h1>
 
         <p>{{ t('Check their address is right. You can correct or update address.') }}</p>
-        <div>
-          <h2 class="heading-medium">{{ t('Full name') }}</h2>
-          <p>{{ claim.claimData.defendant.name }}</p>
+        {% if claim.claimData.defendant.type == 'individual' %}
+          {%
+            set fields = [
+              { label: 'Full name', value: claim.claimData.defendant.name, readOnly: true }
+            ]
+          %}
+        {% elseif claim.claimData.defendant.type == 'soleTrader' %}
+          {%
+            set fields = [
+              { label: 'Full name', value: claim.claimData.defendant.name, readOnly: true },
+              { label: 'Business name', value: claim.claimData.defendant.businessName, readOnly: true }
+            ]
+          %}
+        {% elseif claim.claimData.defendant.type == 'company' %}
+          {%
+            set fields = [
+              { label: 'Company name', value: claim.claimData.defendant.name, readOnly: true },
+              { label: 'Contact person', value: claim.claimData.defendant.contactPerson, readOnly: true }
+            ]
+          %}
+        {% elseif claim.claimData.defendant.type == 'organisation' %}
+          {%
+            set fields = [
+              { label: 'Organisation name', value: claim.claimData.defendant.name, readOnly: true },
+              { label: 'Contact person', value: claim.claimData.defendant.contactPerson, readOnly: true }
+            ]
+          %}
+        {% endif %}
 
-          <h2 class="heading-medium">{{ t('Their address') }}</h2>
-          {{ addressFragment(form) }}
+        {{
+          partyDetailsFragment(
+            form,
+            fields = fields,
+            addressSection = {
+              heading: 'Their address',
+              name: undefined
+            },
+            showCorrespondenceAddressSection = false
+          )
+        }}
 
-          <h2 class="heading-medium">{{ t('Email address') }}</h2>
-          <p>{{ claim.claimData.defendant.email }}</p>
+        {{ readOnlyField(label = 'Email address', value = claim.claimData.defendant.email) }}
```

Comment:
this is left over from before, but I think there should be a `if defined` wrapped around the email block.
Might be worth checking

Reply:
Good reminder, confirmed it with @tarik-johnston1, made it conditional. Will update @pragnesh-patel on this.